### ISSUE TITLE
Doesn't render Todoist-Obsidian plugin priority colors

### ISSUE THREAD

**Issue by anvimea** [kepano/obsidian-minimal#108]
I've dabbled with Minimal for awhile but always reverted to another theme because the [Obsidian-Todoist sync plugin](https://github.com/jamiebrynes7/obsidian-todoist-plugin) doesn't seem to be able to render priority colors in Minimal.

How a Priority 3 - blue - task appears in Minimal:
![Obsidian_M3OCRHkWnY](https://user-images.githubusercontent.com/20266830/148814496-62b0fc26-643a-496e-915a-a9aba7ceec1a.png)

How a Priority 3 - blue - task appears when the color is successfully rendered:
![Obsidian_9Kd9dBP2PB](https://user-images.githubusercontent.com/20266830/148814565-6492f844-1aa1-4b54-bfa6-a70a60949124.png)

I've tested it in an environment with only Minimal & the Todoist plugin installed just to be sure it isn't some broader CSS/plugin conflict.

It's not a big deal at all but I noticed no one had mentioned it in any open or closed issue here. This is perhaps intentional behavior, but if so, is there a way around it?

**Comment by aventuraviper** [comment]
The following snippet will enable this for you. Copy it to a new css file in your snippets directory (Obsidian settings > Appearance > Open snippets folder)

```
/* Todoist Plugin Styles */

.todoist-query-title {
  display: inline !important;
}

.todoist-refresh-button {
  display: inline;
  float: right;
  margin-left: 8px;
  padding: 3px 10px;
  transition: background-color 300ms ease-in-out;
}

.contains-task-list {
	font-size: 14px;
}

.todoist-p1 > input[type="checkbox"] {
    border: 2px solid #ff7066 !important;
    border-radius: 50%;
    background-color: rgba(255, 112, 102, 0.10) !important;
    transition: background-color 300ms ease-in-out;
}

.todoist-p1 > input[type="checkbox"]:hover {
	background-color: rgba(255, 112, 102, 0.25) !important;
}

.todoist-p2 > input[type="checkbox"] {
    border: 2px solid #ff9a14 !important;
    border-radius: 50%;
    background-color: rgba(255, 154, 20, 0.10) !important;
    transition: background-color 300ms ease-in-out;
}

.todoist-p2 > input[type="checkbox"]:hover {
	background-color: rgba(255, 154, 20, 0.25) !important;
}

.todoist-p3 > input[type="checkbox"] {
    border: 2px solid #5297ff !important;
    border-radius: 50%;
    background-color: rgba(82, 151, 255, 0.10) !important;
    transition: background-color 300ms ease-in-out;
}

.todoist-p3 > input[type="checkbox"]:hover {
	background-color: rgba(82, 151, 255, 0.25) !important;
}

.todoist-p4 > input[type="checkbox"] {
    border: 2px solid #808080 !important;
    border-radius: 50%;
    background-color: rgba(128, 128, 128, 0.10) !important;
    transition: background-color 300ms ease-in-out;
}

.todoist-p4 > input[type="checkbox"]:hover {
	background-color: rgba(128, 128, 128, 0.25) !important;
}

.task-metadata {
  font-size: 12px !important;
  display: flex;
  justify-content: space-between;
  margin-left: 1rem !important;
  margin-bottom: 0.5rem;
}

.task-list-item {
	border-bottom: 1px solid rgba(128, 128, 128, 0.25);
	margin-bottom: 0.5rem;
}

.task-list-item:last-child {
	border-bottom: none;
}
```

**Comment by anvimea** [comment]
That indeed does do the trick, though I think I'll have to mess around with the font-size a bit. Thanks a lot!

Will still leave this open since it's been added to the roadmap and perhaps a more native solution is still in the cards. If not, please feel free to close it. @kepano

**Comment by kepano** [comment]
This is now available as of 5.1.8
